spec/system/user_spec.rb

```diff
@@ -10,7 +10,7 @@
         fill_in :user_firstname, with: Faker::Name.first_name
         fill_in :user_lastname, with: Faker::Name.last_name
         fill_in :user_address, with: Faker::Address.full_address
-        fill_in :user_phone_number, with: Faker::PhoneNumber.cell_phone
+        fill_in :user_phone_number, with: '0606060606'
```

Comment:
Thanks for catching that one. I think you can re-use the _french_phone_number_ sequence Factory if you get issues with a static number

```suggestion
        fill_in :user_phone_number, with: FactoryBot.generate(:french_phone_number)
```